Makefile

```diff
@@ -207,12 +207,16 @@ else
   endif
 endif
 
+# Attempt to find and include OpenMPI on the system
+OPENMPI_DIR = /usr/lib/x86_64-linux-gnu/openmpi
```

Comment:
You might want to make this ?= that way you can override it at the command line?

Reply:
Good call